src/core/labeling/qgspallabeling.h

```diff
@@ -636,6 +633,12 @@ class CORE_EXPORT QgsPalLayerSettings
     //! True if label rotation should be preserved during label pin/unpin operations.
     bool preserveRotation = true;
 
+    //! Unit for rotation of labels.
+    QgsUnitTypes::AngleUnit rotationUnit() const;
```

Comment:
```suggestion
    /**
     * Unit for rotation of labels.
     * \see setRotationUnit()
     * \since QGIS 3.22
     */
    QgsUnitTypes::AngleUnit rotationUnit() const;
```